test/integration/feedback_test.lua

```diff
@@ -50,6 +50,10 @@ g.before_all = function()
         },
     })
     g.cluster:start()
+
+    g.msg = ch:get(2)
+    t.assert_type(g.msg, 'table', 'No feedback received')
```

Comment:
Could you check msg content here?

Reply:
assert_type already prints value if type mismatches

```console
tarantool> t = require('luatest')
---
...

tarantool> t.assert_type('foo', 'table', 'something serious is broken')
---
- error:
    status: fail
    class: LuatestError
    message: 'something serious is broken

      expected: a table value, actual: type string, value "foo"'
...
```

And other fields are checked in tests.